app/models/concerns/revision_count.rb

```diff
@@ -0,0 +1,21 @@
+require 'active_support/concern'
+
+module RevisionCount
+  extend ActiveSupport::Concern
+
+  included do
+    after_update :increment_revision
+  end
+
+  def increment_revision
+    conference = if is_a?(Room)
```

Comment:
I reaaaaally don't like checking the type of the class to see what method to call.
Prefer [duck typing](https://robots.thoughtbot.com/back-to-basics-polymorphism-and-ruby#duck-typing)!

We can just set a "requirement" for the `RevisionCount` module: if a class includes this module, it has to implement the `conference` method.
So:
Room could have `has_one :conference, through: :venue`
Conference: `def conference; self; end`
Event: `has_one :conference, through: :program`
That way, the `increment_revision` method is reduced to:
```ruby
def increment_revision
  conference.increment!(:revision)
end
```